### ISSUE TITLE
Installation/Running

### ISSUE THREAD

**Issue by Emyrk** [saml-dev/gome-assistant#10]
I am new to HA and hopefully this is not a dumb question.

Do you just run the Golang application on your own somewhere on your home network? There is no integration that can be installed into HA?

**Comment by saml-dev** [comment]
Correct, you just run it wherever and it connects to HA using the HTTP and Websocket APIs. That's why you have to give the URL of your HA instance when setting up gome-assistant.

**Comment by Emyrk** [comment]
@saml-dev makes sense. HA addons are just docker containers. Would be cool to package it as an HA addon somehow.

**Comment by saml-dev** [comment]
Yeah I've thought about this as it would probably increase the reach of GA, but I doubt I'll ever prioritize my time to figure out. PRs welcome if it's worth it to you 😁

**Comment by Emyrk** [comment]
I ended up just going with Node Red :cry:

**Comment by saml-dev** [comment]
That's where I started too! Hope it works well for you.
